lib/processSketch.js

```diff
@@ -53,4 +54,20 @@ const sortByName = (a, b) => {
 	return a1> b1? 1: -1;
 }
 
+const extractModifiers = (sketch) => {
+    const modifiers = sketch.getPages()[0].layers.filter((layer) => {
+        return layer.name.includes("--")
```

Comment:
Could you add a comment explaining what the `--` logic is for?

Reply:
Modifiers are used for a very specific use case. In fact, we only use them for one reason, and that's to add corner radius to selected layers.

Sketch has chosen to not include Corner Radius into the Layer Style, and this led us to the modifiers. Much like adding an Atom in the theme, a designer can also add a modifier which sets the corner radius.

Example:
If we need to add rounded corners to the `/*/*/*/background` atom, we need to create a new atom named `/*/*/*/background --radius` and include a layer with the rounded corner settings. 

The above also adheres to the overarching concept. You can have a default setting for the Radius `/*/*/*/background --radius` modifier, or you can also target more specific atoms eg. `home/footer/primaryButton[selected]/background --radius`.